runtime_config_linux.go

```diff
@@ -11,9 +11,9 @@ type LinuxRuntimeSpec struct {
 
 type LinuxRuntime struct {
 	// UidMapping specifies user mappings for supporting user namespaces on linux.
-	UidMappings []IDMapping `json:"uidMappings"`
+	UIDMappings []IDMapping `json:"uidMappings"`
 	// UidMapping specifies group mappings for supporting user namespaces on linux.
```

Comment:
On Mon, Aug 31, 2015 at 04:11:40PM -0700, Mrunal Patel wrote:
> >  	// UidMapping specifies group mappings for supporting user namespaces on linux.
> 
> And here.

And UID → GID here too ;).